elements/pfe-cta/src/pfe-cta.scss

```diff
@@ -77,13 +70,10 @@ $LOCAL-VARIABLES: map-deep-merge($LOCAL-VARIABLES, (
   font-weight: pfe-local(FontWeight);
   line-height: pfe-local(LineHeight);
   text-decoration: pfe-local(TextDecoration) !important;
-  
-  // cursor: pointer;
 
   :host([priority]) & {
-    --pf-c--FontSize: #{pfe-local(FontSize--priority)};
+    font-size: #{pfe-local(FontSize--priority)};
```

Comment:
```suggestion
    font-size: pfe-local(FontSize--priority, $fallback: pfe-var(FontSize--md));
```

Reply:
I can remove the interpolation but this line is requesting a new variable since its a different variant than the default style